packages/core/src/index.ts

```diff
@@ -20,6 +20,12 @@ export { wakuLightPush, LightPushCodec } from "./lib/light_push/index.js";
 export * as waku_relay from "./lib/relay/index.js";
 export { wakuRelay, RelayCreateOptions } from "./lib/relay/index.js";
 
+export {
+  ConnectionManager,
+  Options,
+  UpdatedStates,
+} from "./lib/ConnectionManager.js";
```

Comment:
```suggestion
} from "./lib/connection_manager.js";
```

we use snake case for file names